server/protocols/jmap-rfc-8621/src/main/scala/org/apache/james/jmap/mail/Email.scala

```diff
@@ -65,17 +65,25 @@ object Email {
   }
 
   def from(message: (MessageId, Seq[MessageResult])): Try[Email] = {
-    val firstMessage: MessageResult = message._2.head
+    val firstMessage: Try[MessageResult] = message._2
```

Comment:
All the Try since this one could be a in the for comprehension. Would be nicer IMO

Reply:
Nice idea ^^